docs/components/handleCodeSandbox.js

```diff
@@ -78,9 +78,12 @@ const handleCodeSandbox = async ({ code, title }: {| code: string, title: string
       },
       'index.js': {
         content: `import React from "react";
-import { render } from "react-dom";
+import { createRoot } from "react-dom/client";
```

Comment:
Netlify takes care of this under the hood for the docs site, so this was the only place that needed to be updated to the new API